data/helpers.d/network

```diff
@@ -0,0 +1,22 @@
+# Normalize the url path syntax
+# Handle the slash at the beginning of path and its absence at ending
+# Return a normalized url path
+#
+# example: url_path=$(ynh_normalize_url_path $url_path)
+#          ynh_normalize_url_path example -> /example
+#          ynh_normalize_url_path /example -> /example
+#          ynh_normalize_url_path /example/ -> /example
+#
+# usage: ynh_normalize_url_path path_to_normalize
+# | arg: url_path_to_normalize - URL path to normalize before using it
+ynh_normalize_url_path () {
+	path=$1
```

Comment:
I don't know about the coding practice used, but changing a variable named "path" looks dangerous to me (c.f. "PATH"). Also a variable "path" could be defined somewhere in the code and maybe that'd impact it ? (Don't know bash enough) [This post](http://www.kfirlavi.com/blog/2012/11/14/defensive-bash-programming/) recommends to use the keyword `local` for variables local to a function and it would be nice to add it here imho.